src/estimation/diagnostics.jl

```diff
@@ -1,3 +1,13 @@
+function StatsBase.residuals(fpm::FittedPumasModel)
+  # Return the residuals
+  return [residuals(fpm.model, subject, fpm.param, vrandeffs) for (subject, vrandeffs) in zip(fpm.data, fpm.vvrandeffs)]
+end
+function StatsBase.residuals(model::PumasModel, subject, param, vrandeffs)
+  # Calculated the dependent variable distribution objects
+  dist = derived_dv_dist(model, subject, param, (η=vrandeffs,))

```

Comment:
```suggestion
  rtrf = totransform(model.random(param))
  randeffs = TransformVariables.transform(rtrf, vrandeffs)
  dist = derived_dv_dist(model, subject, param, randeffs)
```
and similarly for all the other occurrences